grml-debootstrap

```diff
@@ -1385,6 +1391,26 @@ mount_target() {
 }
 # }}}
 
+# mount the new partition or if it's a directory do nothing at all {{{
+mount_boot_target() {
+  if [ -n "$MOUNT_BOOT" ] ; then
+     einfo "/boot is not on a separate partition, nothing to mount."
+  else
+     if grep -q "$MOUNT_BOOT" /proc/mounts ; then
+        ewarn "$MOUNT_BOOT already mounted, continuing anyway." ; eend 0
+     else
+       if ! [ -d "${MNTPOINT}/boot" ] ; then
+          [ -n "$VIRTUAL" ] || mkdir -p "${MNTPOINT}/boot"
+       fi
```

Comment:
Those three lines feel unnecessary, given that /boot should exist anyway and that you're creating it in line 1406 anyway :)